DigitalLearningSolutions.Data/Services/ActivityService.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace DigitalLearningSolutions.Data.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.TrackingSystem;
+
+    public interface IActivityService
+    {
+        public IEnumerable<MonthOfActivity> GetRecentActivity(int centreId);
+    }
+
+    public class ActivityService : IActivityService
+    {
+        private readonly IActivityDataService activityDataService;
+        private readonly IClockService clockService;
+
+        public ActivityService(IActivityDataService activityDataService, IClockService clockService)
+        {
+            this.activityDataService = activityDataService;
+            this.clockService = clockService;
+        }
+
+        public IEnumerable<MonthOfActivity> GetRecentActivity(int centreId)

```

Comment:
Given in the future we will be passing in some parameter from the filters, would it be worth setting this up so it can work with that and doing any default date logic that will be replaced where this is called?

Reply:
I think the processing of the filter parameters is likely to be done in the service, so for the time being I'm inclined to leave it like this until the parameters are set up in the relevant ticket. It may not even use a DateTime object in the eventual implementation.

Reply:
Fair enough, especially if it is not yet clear what the parameters will be.